apps/language_server/lib/language_server/server.ex

```diff
@@ -789,4 +790,14 @@ defmodule ElixirLS.LanguageServer.Server do
   defp set_project_dir(state, _) do
     state
   end
+
+  defp create_gitignore(%{project_dir: project_dir} = state) do
+    with gitignore_path <- Path.join([project_dir, ".elixir_ls", ".gitignore"]),
+         :ok <- gitignore_path |> Path.dirname() |> File.mkdir(),
+         :ok <- File.write(gitignore_path, "*", [:read]) do
+      state
+    else
+      _ -> state
```

Comment:
I'm not sure what should we do when have a problems while creating a file.

Reply:
I think just logging a warning is good (using `JsonRpc.log_message(:warning, message)`)